courses/utils.py

```diff
@@ -166,3 +166,27 @@ def get_unenrollable_courses(queryset):
         .filter(courseruns__id__in=courseruns_qs.values_list("id", flat=True))
         .distinct()
     )
+
+
+def get_catalog_course_filter(relative_filter=""):
```

Comment:
Do the filters in this method actually align with the filters on our catalog page?

Reply:
I am not sure. Maybe the start date is irrelevant, since we show archived courses. What do you think?

Reply:
So it looks like this is the place where courses get filtered for the catalog `class CourseFilterSet`.
The main criteria is_live and is_enrolable.

Reply:
Do you know what defines "is_enrollable" in this case?

Reply:
So I looked into a bit, and In order to reuse the the existing filtering I would need to refactor a bunch of existing code, and might introduce a bug. I think it would be better to do that refactor in a separate PR, and test it thoroughly.